middleware/auth.ts

```diff
@@ -6,4 +6,7 @@ export default defineNuxtMiddleware((context) => {
    * @todo Redirect the user to main page if token is present in store
    * @hints check what propeties context has
    */
+  if (!context.store.getters.auth) {
```

Comment:
there needs to checks based on route being on login getting redirected to login won't make sense